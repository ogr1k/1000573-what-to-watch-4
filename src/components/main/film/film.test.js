import React from "react";
import renderer from "react-test-renderer";
import Film from "./film.jsx";

const film = {
  name: `Jojo Rabbit`,
  poster: ``
};

it(`Render Film`, () => {
  const tree = renderer
    .create(<Film name={film.name} image={film.poster} onClick={() => {}} onmouseover={() => {}}
      onmouseout={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
