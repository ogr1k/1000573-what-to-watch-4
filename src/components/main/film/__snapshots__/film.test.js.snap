// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Film 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <div
    className="small-movie-card__image"
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  >
    <img
      alt="Jojo Rabbit"
      height="175"
      src=""
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Jojo Rabbit
    </a>
  </h3>
</article>
`;
